stacks.yml

```diff
@@ -277,6 +277,9 @@ admin_frontend:
     EnvVarDmApiUrl: "{{ stacks.api.outputs.URL }}"
     EnvVarDmS3DocumentBucket: "{{ stacks.documents_s3.outputs.Name }}"
```

Comment:
Is the `EnvVarDmS3DocumentBucket` obsolete now? If so it would be good to remove it to avoid clutter.